mobile/src/screens/survey/libs/constants.ts

```diff
@@ -0,0 +1,25 @@
+import { type SurveyInputDto } from '#packages/survey/survey';
+
+const TEXTAREA_MAX_LENGTH = 1000;
+
+const TEXTAREA_ROWS_COUNT = 3;
+
+const DEFAULT_SURVEY_PAYLOAD: SurveyInputDto = {
+  options: [],
+  textarea: '',
```

Comment:
```suggestion
  preferences: [],
  other: '',
```

Reply:
it's ok as for me, just split common styles like - button, buttonOutlined etc., so that it will look like this
```
[styles.button, style === 'solid' && styles.buttonSolid]
```
also remove Survey word from styles names